api/src/client/WebserviceLicenseStatusProcessorClient.test.ts

```diff
@@ -29,70 +32,107 @@ describe("WebserviceLicenseStatusProcessorClient", () => {
     searchLicenseStatus = WebserviceLicenseStatusProcessorClient(stubLicenseStatusClient);
   });
 
-  it("searches for license status on name, zipcode, and license type", async () => {
+  it("searches for license status on name and zipcode and receives NO_ADDRESS_MATCH_ERROR", async () => {
     stubLicenseStatusClient.search.mockResolvedValue([]);
     const nameAndAddress = generateLicenseSearchNameAndAddress({
       name: "Crystal",
       zipCode: "12345",
     });
-    await expect(searchLicenseStatus(nameAndAddress, "Home improvement")).rejects.toEqual(
-      new Error(NO_MATCH_ERROR)
-    );
-    expect(stubLicenseStatusClient.search).toHaveBeenCalledWith("crystal", "12345", "Home improvement");
+    await expect(searchLicenseStatus(nameAndAddress)).rejects.toEqual(new Error(NO_ADDRESS_MATCH_ERROR));
+    expect(stubLicenseStatusClient.search).toHaveBeenCalledWith("crystal", "12345");
```

Comment:
dust - Should be "Crystal" to match parent?

Reply:
[pebble] It gets changed to lowercase before the search happens according to the code but I think for what it's worth, there should be a test addressing that behavior similar to how there is one for removing white spaces

Reply:
Because this functionality is implicitly tested, I'm not writing a dedicated test, but I am updating the following test name to call out this expected behavior.